pylti/flask.py

```diff
@@ -171,14 +174,14 @@ def is_role(self, role):
         log.debug("is_role {}".format(role))
         roles = session['roles']
         if role in LTI_ROLES:
-            list = LTI_ROLES[role]
+            list_of_roles = LTI_ROLES[role]
             log.debug("is_role roles_list={} role={} in list={}"
-                      .format(list, roles, roles in list))
-            return roles in list
+                      .format(list_of_roles, roles, roles in list_of_roles))
+            return roles in list_of_roles
         else:
             raise LTIException("Unknown role {}.".format(role))
 
-    def _check_role(self):
+    def check_role(self):
```

Comment:
@symbolist because of warning
 W:354,16: Access to a protected member _check_role of a client class (protected-access)